Lib/test/pythoninfo.py

```diff
@@ -179,6 +179,9 @@ def collect_platform(info_add):
             info_add(f'platform.freedesktop_os_release[{key}]',
                      os_release[key])
 
+    if sys.platform == 'android':
+        call_func(info_add, 'platform.android_ver', platform, 'android_ver')
```

Comment:
FYI you can just call:

```suggestion
    call_func(info_add, 'platform.android_ver', platform, 'android_ver')
```

It does nothing if the function doesn't exist.

Reply:
Like the other `platform.*_ver` functions, `android_ver` does exist on all platforms, but returns empty values when not running on Android. There's no point in cluttering the report in that case.